src/commands/get_pseudo_random.rs

```diff
@@ -0,0 +1,44 @@
+//! Get Pseudo Random Bytes
+//!
+//! <https://developers.yubico.com/YubiHSM2/Commands/Get_Pseudo_Random.html>
+//!
+use super::{Command, Response};
+use {CommandType, Connector, Session, SessionError};
+
+/// Get some number of bytes of pseudo random data generated on the device
+pub fn get_pseudo_random<C: Connector>(
+    session: &mut Session<C>,
+    bytes: u16,
+) -> Result<Vec<u8>, SessionError> {
+    if bytes >= 2045 {
```

Comment:
It might be worth verifying this is actually the case, and if so giving it a constant